multiples.hs

```diff
@@ -1,2 +1,12 @@
-find3 x = if ( `mod` 3) == 0
-    then 
\ No newline at end of file
+checkmul x = if ( x `mod` 3) == 0
+    then x
+    else if ( x `mod` 5) == 0
+    then x
+    else 0
+
+tolist x = [x]
+findmul [] = [0]
+findmul xs = (tolist (checkmul (head xs))) ++ (findmul (tail xs))
+main :: IO()
+main = do
+    print (sum (findmul [1..1000-1]))
```

Comment:
what is the output of this? Does this answer the core question in the problem statement:

> Find the **sum of all the multiples** of 3 or 5 below 1000.